src/Operation/Explain.php

```diff
@@ -49,26 +51,40 @@ class Explain implements Executable
      *
      * Supported options:
      *
-     *  * verbosity (string): The mode in which the explain command will be run.
+     *  * readPreference (MongoDB\Driver\ReadPreference): Read preference. This
+     *    will be used to select a new server when resuming. Defaults to a
+     *    "primary" read preference.
+     *
+     *  * session (MongoDB\Driver\Session): Client session.
      *
      *  * typeMap (array): Type map for BSON deserialization. This will be used
      *    used for the returned command result document.
      *
+     *  * verbosity (string): The mode in which the explain command will be run.
+     *
      * @param string $databaseName      Database name
      * @param Explainable $explainable  Operation to explain
      * @param array  $options           Command options
      * @throws InvalidArgumentException for parameter/option parsing errors
      */
     public function __construct($databaseName, Explainable $explainable, array $options = [])
     {
-        if (isset($options['verbosity']) && ! is_string($options['verbosity'])) {
-            throw InvalidArgumentException::invalidType('"verbosity" option', $options['verbosity'], 'string');
+        if (isset($options['readPreference']) && ! $options['readPreference'] instanceof ReadPreference) {
+            throw InvalidArgumentException::invalidType('"readPreference" option', $options['readPreference'], 'MongoDB\Driver\ReadPreference');
+        }
+
+        if (isset($options['session']) && ! $options['session'] instanceof Session) {
```

Comment:
Can you add tests for both of these new options to the ExplainTest? Should just be a copy/paste from another Operation test. Feel free to amend to the same commit in which you're adding this code.